scripts/generate_index_files.py

```diff
@@ -105,10 +111,16 @@ def curate_tad_elements(tad_df, input_df, gen_class):
                                     ignore_index=True)
             bnd_df = bnd_df.append(o_df, ignore_index=True)
 
-        # Assign TAD information
-        elem_sub_df['TAD_id'] = tad_id
-        elem_sub_df['TAD_start'] = start
-        elem_sub_df['TAD_end'] = end
+        if gen_class == 'LD':
```

Comment:
Perhaps or that each TAD should just have a section in a single notebook. Just that the point of a script with arguments is to flexibly perform a certain analysis. This level of hardcoding makes me wonder why you accept arguments at all?

I think the best time to address these issues would be if you're creating a new repo and we brainstorm the analysis layout. Not really worth retrospectively modifying this.